lib/fa2/common/tzip12.datatypes.jsligo

```diff
@@ -0,0 +1,37 @@
+export type tokenMetadataData = {
+    token_id: nat,
+    token_info: map<string, bytes>
+};
+
+export type tokenMetadata = big_map<nat, tokenMetadataData>;
+
+// Transfer entrypoint
+
+export type atomic_trans = // @layout comb
```

Comment:
@layout comb should be deleted, it's the default one in Ligo now.
Same for each occurence of @layout comb in this project

Reply:
old legacy code removal ongoing ..